### PR TITLE
[Feature]Support time interval for flink connector

### LINKED ISSUE #5959: Support time interval for flink connector
If the current flink connector needs real-time synchronization, sink.batch.size=1 must be set, but when this is set to a large amount of data, the streamload will drop into the library one by one, and the performance is extremely poor.

So refer to the settings of flink-jdbc-connector, increase the time interval option of streamload, after the time has passed, the asynchronous thread will write the data to the doris

#5934

### PR DESCRIPTION
**Pull request by JNSimba**
## Proposed changes

add **sink.batch.interval** option，support flink write regularly

resolve #5959 

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by EmmyMiao87** [comment]
Please add an issue first ~

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ public void load(String value) throws StreamLoadException {
 
     private LoadResponse loadBatch(String value) {
         Calendar calendar = Calendar.getInstance();
-        String label = String.format("audit_%s%02d%02d_%02d%02d%02d_%s",
+        String label = String.format("doris_sink_%s%02d%02d_%02d%02d%02d_%s",
```
Comment (EmmyMiao87) [inline extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java:137]: It is better to have prefixes similar to those related to Flink.

**Comment by EmmyMiao87** [comment]
@hf200012

**Comment by hf200012** [comment]
> @hf200012 
> 
> 

Review, no problem

File: extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java (hunk 1 above)
Comment (hf200012) [reply extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java]: It is recommended to change to doris_flink_sink_%s%02d%02d_%02d%02d%02d_%s

**Comment by morningman** [comment]
Hi @JNSimba Please resolve the conflict

**Comment by JNSimba** [comment]
> Hi @JNSimba Please resolve the conflict

resolved

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**